README.md

```diff
@@ -35,7 +35,10 @@ That's why I made this fork:
 
 Add the lib using composer:
 
-    composer require j0k3r/graby
+    composer require j0k3r/graby php-http/guzzle5-adapter
+
+Why `php-http/guzzle6-adapter`? Because Graby is decoupled form any HTTP messaging client with help by [HTTPlug](http://httplug.io/).
```

Comment:
Shouldn't this be `php-http/guzzle5-adapter`?
Or the line above is wrong and should be `php-http/guzzle6-adapter`